ui/app/mixins/cluster-route.js

```diff
@@ -9,6 +9,7 @@ const CLUSTER = 'vault.cluster';
 const CLUSTER_INDEX = 'vault.cluster.index';
 const OIDC_CALLBACK = 'vault.cluster.oidc-callback';
 const DR_REPLICATION_SECONDARY = 'vault.cluster.replication-dr-promote';
+const DR_REPLICATION_SECONDARY_DETAILS = 'vault.cluster.replication-dr-promote.details';
```

Comment:
looks like this variable isn't use which is causing test failures. once it's removed this is 💯 !

Reply:
Good catch. However, I am reverting back to using this variable (see thread with Matthew) so the tests should pass (I double checked 🙂).